mlos_bench/mlos_bench/tests/tunables/tunables_assign_test.py

```diff
@@ -162,3 +162,37 @@ def test_tunable_assign_null_to_float(tunable_float: Tunable) -> None:
         tunable_float.value = None
     with pytest.raises(TypeError):
         tunable_float.numerical_value = None    # type: ignore[assignment]
+
+
+def test_tunable_assign_special(tunable_int: Tunable) -> None:
+    """
+    Check the assignment of a special value outside of the range (but declared `special`).
+    """
+    tunable_int.numerical_value = -1
+    assert tunable_int.numerical_value == -1
```

Comment:
need an assert for `is_special` too?

Reply:
`Tunable` does not have an explicit `is_special` property, but I can make one